src/screens/SignUpScreen/index.js

```diff
@@ -30,49 +31,79 @@ const SignUpScreen = ({ history }) => {
           password,
           firstName,
           lastName,
-          quarantineLocation,
-          times,
           venmo,
           gender,
           lookingFor,
-          session: currentSession.id
+          friendEmails,
+          session: nextSession
         });
+        history.push("/");
       } catch (err) {
-        console.error("Error: ", err.message);
+        setServerError(err.message);
+        window.scrollTo(0, 0);
+        console.error("Encountered error signing up: " + err.message);
       }
       setSubmitting(false);
-      history.push("/");
     },
-    [history, currentSession]
+    [history, nextSession, setServerError]
   );
 
-  const validateForm = useCallback(({ email, password, passwordConfirm }) => {
-    const errors = {};
-    if (!email) {
-      errors.email = "Required";
-    } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i.test(email)) {
-      errors.email = "Invalid email address";
-    }
-    if (!password) {
-      errors.password = "Required";
-    } else if (password.length < 8) {
-      errors.password = "Must be 8 characters long";
-    }
-    if (!passwordConfirm) errors.passwordConfirm = "Required";
-    else if (passwordConfirm !== password)
-      errors.passwordConfirm = "Passwords do not match";
-    return errors;
-  }, []);
+  const validateForm = useCallback(
+    ({
+      email,
+      firstName,
+      lastName,
+      password,
+      gender,
+      passwordConfirm,
+      venmo,
+      lookingFor,
+      friendEmails
+    }) => {
+      const errors = {};
+
+      if (!email) errors.email = "Required";
+      else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i.test(email))
+        errors.email = "Invalid email address";
+
+      if (!firstName) errors.firstName = "Required";
+      if (!lastName) errors.lastName = "Required";
+      if (!venmo) errors.venmo = "Required";
```

Comment:
I think venmo doesn't have to be required. I can't even create a damn account because I don't have a US number :(